Gemfile

```diff
@@ -65,6 +64,8 @@ gem "sass-rails", "~> 6.0"
 gem "scenic", "~> 1.5.1"
 gem "scout_apm", "~> 2.5"
 gem "screenshot_machine", "~> 0.0.4", github: "coderberry/screenshot_machine"
+gem "sendgrid-actionmailer", "~> 3.0"
+gem "sendgrid-ruby", "~> 6.2"
```

Comment:
This is a dependency of the above gem, do we need both?

Reply:
Good call. I'll remove it